docs/conf.py

```diff
@@ -48,16 +48,16 @@
 
 # General information about the project.
 project = u'libCellML'
-copyright = u'2014, David Nickerson'
+copyright = u'2014-2015, University of Auckland'
```

Comment:
This could be an issue for external contributors. (All my work is officially owned by Oxford.)

Not that we've managed to get research services here to sort out a proper contributor licence agreement for Chaste, having been asking for >5 years, but you may have more luck at Auckland!